docs/topic_guides/evaluation/model.md

```diff
@@ -1,3 +1,33 @@
 # Model Evaluation
 
-This page is under construction - check back soon!
\ No newline at end of file
+The parameters of a trained Splink model build up a set of rules for scoring any pairwise records comparisons. Before scoring any pairs of records there are a number of ways to check whether your model will perform as you expect.
```

Comment:
Suggestion to tweak this slightly to something like the below. To me, the parameters don't so much build the rules themselves as feature in them. I see the rules more as the relationship between the gamma value and trained param partly determined by the comparisons levels, i.e. gamma=3 -> params=blah -> weight = 14, etc

```suggestion
The parameters in a trained Splink model determine the match probability (Splink score) assigned to pairwise record comparisons. Before scoring any pairs of records there are a number of ways to check whether your model will perform as you expect.
```
Apologies if this is overly pedantic!